packages/terra-form-radio/tests/jest/RadioField.test.jsx

```diff
@@ -76,3 +77,20 @@ it('correctly applies the theme context className', () => {
   );
   expect(wrapper).toMatchSnapshot();
 });
+
+it('correctly applies "inputAttrs" property to the Radio component', () => {
+  const attrKey = 'data-custom-attr';
+  const attrValue = 'attr data';
+  const radioField = (
+    <RadioField legend="Default RadioField">
+      <Radio
+        labelText="Default label"
+        inputAttrs={{
+          [attrKey]: attrValue,
+        }}
+      />
+    </RadioField>
+  );
+  const wrapper = mountWithIntl(radioField);
+  expect(wrapper.find('input').prop(attrKey)).toBe(attrValue);
```

Comment:
```suggestion
  expect(wrapper.find('input').prop(attrKey)).toBe(attrValue);
  expect(wrapper).toMatchSnapshot();
```
I would suggest to include snapshot as well !! this would help to validate if attributes are overlapped !!